apps/frontend/src/pages/auth/EmailVerification.tsx

```
@@ -0,0 +1,242 @@
+                                    }
+                                </Button>
+
+                                <Button asChild variant="outline" className="w-full">
+                                    <Link to="/login">Back to login</Link>
+                                </Button>
+                            </div>
+                        </CardContent>
+                    </Card>
+                );
+
+            case 'error':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)